pkgs/applications/editors/nano/test-with-expect.nix

```diff
@@ -0,0 +1,34 @@
+{ nano, expect, runCommand, writeScriptBin }:
+
+let expectScript = writeScriptBin "expect-script" ''
+  #!${expect}/bin/expect -f
+
+  # Load nano
+  spawn nano /tmp/file.txt
```

Comment:
You should use `$TMPDIR` (or even `mktemp -d`) instead of expecting `/tmp` to exist.

Reply:
eh I guess I'll just use cwd, thanks for pointing it out.

Reply:
eh I guess I'll just use cwd, thanks for pointing it out.